pandas/tests/plotting/frame/test_frame.py

```diff
@@ -156,8 +156,8 @@ def test_nullable_int_plot(self):
                 "A": [1, 2, 3, 4, 5],
                 "B": [1.0, 2.0, 3.0, 4.0, 5.0],
                 "C": [7, 5, np.nan, 3, 2],
-                "D": pd.to_datetime(dates, format="%Y"),
-                "E": pd.to_datetime(dates, format="%Y", utc=True),
+                "D": pd.to_datetime(dates, format="%Y").view("i8"),
```

Comment:
why this change?

Reply:
bc we pass `dtype=np.int64` to the DataFrame constructor (seeing this motivated #38842) which casts these two columns, which is deprecated